### PR TITLE
bug fix: remove appended '__' from atb value for calculations

### PR DESCRIPTION
**Pull request by alohaas**
This fixes the NaN issue that was defaulting surveys to DOC_15.

### CODE REVIEW

File: lib/ddg-atbgen.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ exports.calc = {
     },
     atbDelta: function(atb) {
         var ogMajorVersion = atb.split('-')[0].slice(1),
-            ogMinorVersion = atb.split('-')[1],
+            ogMinorVersion = atb.split('-')[1].slice(0, 1),
```
Comment (jdorweiler) [inline lib/ddg-atbgen.js]: The minor version is always one character?

Comment (alohaas) [inline lib/ddg-atbgen.js]: Yeah it's always one character. I'm changing this to a regex though.

Comment (alohaas) [inline lib/ddg-atbgen.js]: Safer...

**Comment by jdorweiler** [comment]
👍

**Pull request merged**